app/views/pets/index.html.erb

```diff
@@ -3,12 +3,15 @@
   <li><%= link_to "All Shelters", "/shelters" %></li>
 </ul>
 <h1 class="title">Pets</h1>
-
+<ul class="menu">
+  <li><%= link_to "Adoptable Pets", "/pets?adoptable=true" %></li>
+  <li><%= link_to "Pets Pending Adoption", "/pets?adoptable=false" %></li>
+</ul>
```

Comment:
adoption filtering links are here in pets index, styling is minimal at this point